### PR TITLE
Updated Doc Strings 

### PR DESCRIPTION
**Pull request by zytegalaxy**
Took a first pass at updating doc strings. README still needs to be updated and set of demos and examples should be added to explain the use cases for offered widgets.

### CODE REVIEW

File: ubo_gui/__init__.py [hunk 1]
```diff
@@ -1,3 +1,7 @@
+"""
+This module initializes the ubo_gui package and registers various widgets with the Kivy Factory.
```
Comment (sassanh) [inline ubo_gui/__init__.py:2]:
```suggestion
This module initializes the `ubo_gui` package and registers various widgets with the Kivy Factory.
```

We need to put all code-related/machine-formatted elements in backtick, it includes all variable names, function names, module names, package names, programming language keywords, paths, repository names. Modern code editors all render these backtick elements correctly as code (like in how they are rendered in markdown) like this:

image

**Comment by zytegalaxy** [comment]
@sassanh I had to do a rebase which I don't know if It was a good idea. I updated the docs and added backticks. We can further improve the docs but perhaps we can get this PR merged...

**Comment by sassanh** [comment]
@mehrdadfeller I checked some files, it looks good. A full review would be very time consuming and considering our priorities, I can't manage to do it. I say lets merge it and then improve it over time if we find any issues.

File: ubo_gui/animated_slider/__init__.py [hunk 2]
```diff
@@ -1,3 +1,5 @@
+"""Module for the AnimatedSlider class."""
```
Comment (sassanh) [inline ubo_gui/animated_slider/__init__.py]:
```suggestion
"""Module for the `AnimatedSlider` class."""
```

You missed this one.

**Review by sassanh — APPROVED** [review]

**Comment by mehrdadfeller** [comment]
@sassanh no merge conflict :) all changes were on docstrings and no heavy review was needed. just wanted to get your feedback before merge

**Pull request merged**